xbmc/utils/LangCodeExpander.cpp

```diff
@@ -8,6 +8,10 @@
 
 #include "LangCodeExpander.h"
 
+#include "LangInfo.h"
+#include "ServiceBroker.h"
```

Comment:
This also needs further thinking. Including references to service broker and add-on manager in an utils class doesn't look right. I.e. this is not some isolated component but rather something that can be called from any component

Reply:
I will find an alternative way both classes are intrinsic

Reply:
Yeah I'll also try to think a bit about it, including such headers here creates unwanted dependencies between components.
I think this calls more and more to a resources component that can be the owner of all static resources (fonts, language, colours, etc). This is too much work thought so if you can find a way to self contain this in the meanwhile it'd be great already.

Reply:
i removed the other includes but `LangInfo.h` i think cannot be removed without a large refactor

Reply:
I think it's totally fine for now, both Langinfo and languagecodeexpander are some sort of components (although accessed as globals) which clearly urges for a refactoring (and inherently to my comment above regarding resources).
Unfortunately it's quite a huge amount of work.
Nevertheless, thanks much for your patch